lecture-notes/3-bld.md

```diff
@@ -896,6 +908,8 @@ you.
 
 ### Compilation
```

Comment:
Let's frame this section/lecture by explicitly stating that we've been talking about `.o` files and linking without really diving into what happens under the hood, and that this section's purpose is to go back and explain that rather than to introduce new build system concepts.